src/components/Organization/Dashboard/ProcessCard.tsx

```diff
@@ -11,13 +16,76 @@ const ProcessCard = () => {
   if (!election) return null
 
   return (
-    <Link as={RouterLink} to={`/processes/${ensure0x(election.id)}`} w='full'>
+    <Link as={RouterLink} to={`/processes/${ensure0x(election.id)}`} w='full' textDecoration='none'>
       <ContentsBox w='full'>
-        <ElectionTitle fontSize='md' textAlign='left' />
-        <ElectionSchedule fontSize='xs' textAlign='left' />
+        <Flex
+          flexDirection={{ base: 'column', lg: 'row' }}
+          alignItems={{ base: 'start', lg: 'center' }}
+          justifyContent='space-between'
+        >
+          <ElectionStatusBadge fontSize='text-sm' mb={{ base: 3, lg: 0 }} order={{ lg: '2' }} />
+          <ElectionTitle fontSize='text' textAlign='left' fontWeight='extrabold' order={{ lg: '1' }} />
+        </Flex>
+        <HStack mb={3}>
+          <MdHowToVote />
+          <Text fontWeight='600' fontSize='text-sm' color='dashboard_card_text' className='dashborad_align_card_text'>
+            {(election as any).voteCount} ({((election as any).voteCount / (election as any).census.size) * 100}%) votes
+            submited
+          </Text>
+        </HStack>
+        <HStack>
+          <IoIosCalendar />
+          <HStack>
+            <Text fontWeight='600' fontSize='text-sm' className='dashborad_align_card_text'>
+              <Text as='span' color='gray'>
+                {(election as any).startDate.getTime() < new Date().getTime() ? (
```

Comment:
Same, you should not be using as any nowhere.

If the problem is because you cannot use `.startDate` on an InvalidElection, start by adding a `if (election instanceof InvalidElection) return null` at the beginning of the component.